### ISSUE TITLE
Flags for depracated versions?

### ISSUE THREAD

**Issue by acesnik** [conda-forge/dotnet-feedstock#59]
Is there any way to tag dotnetcore2.1 as depracated in conda? There are some others that are coming up on end of support, too.

**Comment by dhirschfeld** [comment]
I don't believe there is, but then I also don't think it's the packages responsibility to say whether or not the code is deprecated. 

A `mamba update --all` will update to the latest & greatest unless the user has explicitly pinned the package, in which case I think it's probably the users' responsibility to ensure they're not using deprecated versions.

So, I think I fall on the User Beware side of the line, but even if I didn't, I don't think there's anything we can do short of marking the package as broken which might be undesirable as a user who knows what they're doing might have a valid reason to install an old version.

**Comment by dhirschfeld** [comment]
The folks on the `conda-forge` gitter might be able to say more if it's a concern:
https://gitter.im/conda-forge/conda-forge.github.io
